src/components/Header/SideNav/index.js

```diff
@@ -47,9 +48,12 @@ const SideNavItem = ({ item, path }) => {
   } else {
     const { label, to, href } = item
     return (
-      <OuterAnchorItem activeClassName="active" to={to} href={href}>
-        {label}
-      </OuterAnchorItem>
+      <OuterAnchorItem
```

Comment:
In the old version you have <Component>{label}</Component>
In the new version that {label} in the middle is missing - don't we need it to actually disable the content of the label to the user?

Reply:
`{label}` was a child of this component which isn't necessary, it can be included as a props as that is clearer as to what its getting at in the `OuterAnchorItem` component definition. 

It's a bit of personal preference and style tbh, I was playing around with this component and ended up putting it back together like this.